R/db-interface.R

```diff
@@ -218,11 +232,25 @@ copy_dm_to <- function(dest, dm, ...,
     top_level_fun = "copy_dm_to"
   )
 
+
   # populate tables
-  pwalk(
-    queries[c("name", "remote_name")],
-    ticker_populate(~ db_append_table(dest_con, .y, dm[[.x]], progress))
-  )
+  for (i in seq_along(queries$name)) {
```

Comment:
Good catch, but I'd prefer ignoring this problem for now. It's fine if we can't use `copy_dm_to()` with a dm with data and autoincrement keys. We can resort to `rows_append()` once we have that, or figure out something even nicer.